dt-core/global-functions.php

```diff
@@ -190,7 +190,7 @@ function dt_recursive_sanitize_array( array $array ) : array {
     if ( ! function_exists( 'dt_get_translations' ) ) {
         function dt_get_translations() {
             require_once( ABSPATH . 'wp-admin/includes/translation-install.php' );
-            $translations = wp_get_available_translations(); // @todo throwing errors if wp.org connection isn't established
+//            $translations = wp_get_available_translations(); // @todo throwing errors if wp.org connection isn't established
```

Comment:
This breaks the language selector

Reply:
Sorry. This was a slip. I'm having to disable this line on my development server because it is lagging every page load because it is trying to reach out to Wordpress.org and is being rejected. Basically at coffee shops or other places not at my house. I haven't looked into why. This was an unintentional inclusion.

Reply:
Great find actually. I didn't realize it was being called on every request. 
Worth looking in to.